README.md

```diff
@@ -47,6 +30,14 @@ if (!result.wasSuccessful()) {
 
 ### Building with Maven
 
+```xml
+<dependency>
+  <groupId>io.fusionauth</groupId>
+  <artifactId>fusionauth-java-client</artifactId>
+  <version>1.5.0</version>
```

Comment:
```suggestion
  <version>${fusionauth.version}</version>
```

Reply:
This won't work. But We can make this a managed version file. See `versioned_files` in `fusionauth-ij/fusionauth.yaml`. If we add it to this config, we'll update this when we run `is version`.